README.md

```diff
@@ -63,7 +63,28 @@ print(output.shape)
 
 ## Getting Started
 
-### Building from source (Recommended)
+### Local development
+
+#### Using uv (Recommended)
+
+It’s recommended to use [uv](https://docs.astral.sh/uv/), a very fast Python environment manager, to create and manage Python environments.
+Please follow the documentation to install [uv](https://docs.astral.sh/uv/getting-started/installation/). After installing uv, you can create a new Python environment and install hip-attention using the following commands:
+
+```bash
+# Clone this repository
+git clone git@github.com:DeepAuto-AI/hip-attention.git
+cd hip-attention
+
+# This install all dev dependencies in .venv/
+uv sync
```

Comment:
Now just a single `uv sync` command install everything, including sglang and research